spyder/plugins/editor.py

```diff
@@ -1041,12 +1051,13 @@ def get_plugin_actions(self):
           
         # ---- Run menu/toolbar construction ----
         run_menu_actions = [run_action, run_cell_action,
-                            run_cell_advance_action, MENU_SEPARATOR,
+                            run_cell_advance_action,
+                            re_run_last_cell_action, MENU_SEPARATOR,
                             run_selected_action, re_run_action,
                             configure_action, MENU_SEPARATOR]
         self.main.run_menu_actions += run_menu_actions
         run_toolbar_actions = [run_action, run_cell_action,
-                               run_cell_advance_action, run_selected_action,
+                               run_cell_advance_action,

```

Comment:
Sorry for the confusion, please restore `run_selected_action` here. It was fine before :-)